lib/Doctrine/Common/Cache/PredisCache.php

```diff
@@ -52,6 +52,34 @@ protected function doFetchMultiple(array $keys)
     /**
      * {@inheritdoc}
      */
+    protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
+    {
+        $success = true;
+
+        if (!$lifetime) {
+            // No lifetime, use MSET
+            $response = $this->client->mset(array_map(function ($value) {
+                return serialize($value);
+            }, $keysAndValues));
+
+            $success = ((string) $response === 'OK');
```

Comment:
If the response is `OK`, then there is no need for a cast (strict comparison is already dealing with that)